app/javascript/reducer.js

```diff
@@ -0,0 +1,17 @@
+import { FETCH_GREETING_SUCCESS } from './actions';
```

Comment:
- [ ] [OPTIONAL] I highly recommend creating a folder named redux and moving the store and reducers there so your file structure follows the `duck pattern`